src/instaskip/console_log.clj

```diff
@@ -0,0 +1,10 @@
+(ns instaskip.console-log
+  (:require [clojure.string :as str]
+            [clojure.stacktrace :as cs]))
+
+(defn info [& args] (println (str/join " " args)))
+
+(defn error [ex & args]
+  (println (str "ERROR: Something went wrong: "
```

Comment:
right now I'm filtering what gets to stderr, as the experimental version of clojure gives some warnings :(